### ISSUE TITLE
Release code for Bayesian Approach to Segmentation with Noisy Labels

### ISSUE THREAD

**Issue by NielsRogge** [pfnet-research/Bayesian_SpatialCorr#1]
Hi @t2kasa 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability.If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I noticed that you mentioned that code will soon be available at https://github.com/pfnet-research/Bayesian_SpatialCorr. Let me know once it is available, it'd be awesome to have a code repository that can be linked to the paper on the Hub for improved discoverability. If you plan to release pre-trained model checkpoints or datasets, we'd be happy to host those as well to improve visibility.

Cheers,

Niels
